caddyhttp/httpserver/plugin.go

```diff
@@ -488,6 +488,7 @@ var directives = []string{
 	"markdown",
 	"templates",
 	"browse",
+	"request_id",
```

Comment:
Move this _way_ higher in the chain (like, after "shutdown"), and I'm thinking about whether this should be renamed to "requestid".